Makefile

```diff
@@ -52,8 +52,11 @@ all: manager
 FORCE:
 
 # Run tests
+ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 test: generate fmt vet manifests
-	go test ./... -coverprofile cover.out
+	mkdir -p ${ENVTEST_ASSETS_DIR}
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh
+	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
```

Comment:
Just quote usage, and consistency for variables expanding; as it is a Makefile variable, suggesting to expanding it in the makefile way. Such as:

```raw
	mkdir -p "$(ENVTEST_ASSETS_DIR)"
	test -f "$(ENVTEST_ASSETS_DIR)/setup-envtest.sh" || curl -sSLo "$(ENVTEST_ASSETS_DIR)/setup-envtest.sh" https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh
	source "$(ENVTEST_ASSETS_DIR)/setup-envtest.sh"; fetch_envtest_tools "$(ENVTEST_ASSETS_DIR)"; setup_envtest_env "$(ENVTEST_ASSETS_DIR)"; go test ./... -coverprofile cover.out
```

Reply:
Thanks, I've applied your suggestion.